rats-tls/src/verifiers/api/enclave_verifier_register.c

```diff
@@ -18,16 +18,6 @@ enclave_verifier_err_t enclave_verifier_register(const enclave_verifier_opts_t *
 
 	RTLS_DEBUG("registering the enclave verifier '%s' ...\n", opts->name);
 
-	if (opts->flags & ENCLAVE_VERIFIER_OPTS_FLAGS_SGX2_ENCLAVE) {
```

Comment:
I think this flag bit should be unset as well:

src/verifiers/sgx-ecdsa/main.c:22:	.flags = ENCLAVE_VERIFIER_OPTS_FLAGS_SGX2_ENCLAVE,

Reply:
There are several similar flag for sgx_ecdsa, sgx_ecdsa_qve, sgx_la etc. 
Keep the flag and don't put the flag value as null is to keep the same init styles. 
And it can help future changes or judgement for some redefine the meaning of ENCLAVE_VERIFIER_OPTS_FLAGS_SGX2_ENCLAVE in case.

Reply:
If it don't have actual use, it should be removed. I ask another question: do these flags are useful for sgx mode? If not, please remove all flags for verifiers.